<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GamesW</class>
 <widget class="QDialog" name="GamesW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1247</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Portal</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#GamesW{
	border-image: url(:/imagini/spacebck.jpg);
}</string>
  </property>
  <widget class="QLabel" name="label_mwuser">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>111</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_mwusername">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_mwusername{
color:black;

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="GAMESgb">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>271</width>
     <height>521</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#GAMESgb{
color:white;
background:rgb(255, 48, 155);
border-radius:8px;
}</string>
   </property>
   <property name="title">
    <string>GAMES</string>
   </property>
   <widget class="QPushButton" name="pb_joc1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pb_joc1{
color:white;
background-color:rgb(0, 0, 127);
border-width:0px;
border-radius:4px;
}
#pb_joc1:hover{
	background-color: rgb(255, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>Space Invaders</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_joc2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pb_joc2{
color:white;
background-color:rgb(0, 0, 127);
border-width:0px;
border-radius:4px;
}
#pb_joc2:hover{
background-color: rgb(255, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>Flappy bird</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_joc3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pb_joc3{
color:white;
background-color:rgb(0, 0, 127);
border-width:0px;
border-radius:4px;
}
#pb_joc3:hover{
background-color: rgb(255, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>Snake</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_joc4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pb_joc4{
color:white;
background-color:rgb(0, 0, 127);
border-width:0px;
border-radius:4px;
}
#pb_joc4:hover{
background-color: rgb(255, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>Super Mario</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_joc5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pb_joc5{
color:white;
background-color:rgb(0, 0, 127);
border-width:0px;
border-radius:4px;
}
#pb_joc5:hover{
background-color: rgb(255, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>Sudoku</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_joc6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>440</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pb_joc6{
color:white;
background-color:rgb(0, 0, 127);
border-width:0px;
border-radius:4px;
}
#pb_joc6:hover{
background-color: rgb(255, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>Tetris</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="FRIENDSgb">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>20</y>
     <width>301</width>
     <height>661</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#FRIENDSgb{
color:white;
background:rgb(255, 56, 179);
border-radius:8px;
}</string>
   </property>
   <property name="title">
    <string>MY FRIENDS</string>
   </property>
   <widget class="QListWidget" name="friendslist">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>211</width>
      <height>531</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#friendslist{
color:white;
background:rgb(255, 170, 127);
border-radius:8px;
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_search">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>560</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#lineEdit_search{
color:black;
background:rgb(255, 170, 127);
border-radius:8px;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_button">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>580</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#delete_button{
color:white;
background-color: rgb(170, 0, 0);
border-width:0px;
border-radius:4px;
}
#delete_button:hover{
background-color:#27a9e3;
}</string>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_button">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>560</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#add_button{
color:white;
background-color:#66c011;
border-width:0px;
border-radius:4px;
}
#add_button:hover{
background-color:#27a9e3;
}</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>610</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="infogb">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>560</y>
     <width>581</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#infogb{
color:white;
background:rgb(255, 134, 209);
border-radius:8px;
}</string>
   </property>
   <property name="title">
    <string>Current Information</string>
   </property>
   <widget class="QLabel" name="label_descriere">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>541</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_descriere{

color:white;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="option_gb">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>591</width>
     <height>511</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#option_gb{
color:white;

border-radius:8px;
}</string>
   </property>
   <property name="title">
    <string>Your option</string>
   </property>
   <widget class="QPushButton" name="pb_info">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>440</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#pb_info{
color:white;
background-color:rgb(0, 0, 127);
border-width:0px;
border-radius:4px;
}
#pb_info:hover{
background-color: rgb(255, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>Info</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_play">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>440</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#pb_play{
color:white;
background-color:rgb(0, 0, 127);
border-width:0px;
border-radius:4px;
}
#pb_play:hover{
background-color: rgb(255, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_top">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>440</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#pb_top{
color:white;
background-color:rgb(0, 0, 127);
border-width:0px;
border-radius:4px;
}
#pb_top:hover{
background-color: rgb(255, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>Top 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_currentgame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>561</width>
      <height>371</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="aboutpb">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>640</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#aboutpb{
color:white;
background-color:rgb(0, 0, 127);
border-width:0px;
border-radius:4px;
}
#aboutpb:hover{
background-color: rgb(255, 0, 0);
}</string>
   </property>
   <property name="text">
    <string>About Us</string>
   </property>
  </widget>
  <widget class="QPushButton" name="contactpb">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>640</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#contactpb{
color:white;
background-color:rgb(0, 0, 127);
border-width:0px;
border-radius:4px;
}
#contactpb:hover{
background-color: rgb(255, 0, 0);
}</string>
   </property>
   <property name="text">
    <string>Contact Info</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
